ws-nextjs-app/pages/[service]/send/[id]/FormField/InvalidMessageBox.tsx

```diff
@@ -10,11 +26,19 @@ export default ({ id, messageCode }: InvalidMessageBoxProps) => {
   const message = ugc[messageCode ?? InvalidMessageCodes.FieldRequired];
 
   return (
-    <div>
-      <p id={id}>
-        <span>❗</span>
-        {message}
-      </p>
-    </div>
+    <>
+      <div css={styles.errorArrow} />
+      <div css={styles.errorMessageBox}>
+        <ErrorSymbol />
+        <Paragraph
+          id={id}
+          css={styles.errorText}
+          fontVariant="sansBold"
+          size="minion"
+        >
+          {message}
```

Comment:
I notice this 'message' isn't always returning something. E.g. on Storybook if select 'arabic'. I feel like we should have a fallback here? 

![Screenshot 2024-06-18 at 10 25 49](https://github.com/bbc/simorgh/assets/98817636/d1c3c7f1-f2c7-4b31-8f02-6b2abf3ca159)